memecity.game/Game/States/GameOverState.h

```diff
@@ -0,0 +1,24 @@
+#ifndef _GAME_OVER_STATE_H
+#define __OVER_STATE_H
+#include <Engine/State.h>
+#include <UI.h>
+#include "..\..\GameManager.h"
+
+class GameOverState : public memecity::engine::state::State
+{
+private:
+	GameManager::GameContext* _context;
+	std::unique_ptr<memecity::engine::ui::menu::Menu> menu;
```

Comment:
waarom is dit een unique ptr?

Reply:
dit komt omdat de menu builder een unique_pointer terug geeft.